is_core/rest/resource.py

```diff
@@ -233,7 +233,15 @@ def get_default_fields(self, obj=None):
         return self.core.get_rest_default_fields(self.request, obj=None) if default_fields is None else default_fields
 
     def get_default_fields_rfs(self, obj=None):
-        return super(RESTModelResource, self).get_default_fields_rfs(obj=obj).join(rfs(self.default_fields_extension))
+        return super(RESTModelResource, self).get_default_fields_rfs(obj=obj).join(
+            rfs(self.get_default_fields_extension(obj))
```

Comment:
RESTFieldset, je to datová struktura, která obsahuje strom field a subfield tedy třeba order(customer(first_name, last_name)) výhoda této struktury je že se umí joinovat 

máš 2 rfs
order(customer(first_name),first_name) + order(customer(last_name)) = order(customer(first_name, last_name), first_name)